src/main/kotlin/de/europace/gradle/docker/publish/DockerPublishPlugin.kt

```diff
@@ -20,7 +21,7 @@ class DockerPublishPlugin : Plugin<Project> {
     val dockerBuildContextDir = File("${project.buildDir.path}/docker")
     val extension = project.extensions.findByName(EXTENSION_NAME) as? DockerPublishExtension ?: project.extensions.create(EXTENSION_NAME, DockerPublishExtension::class.java, project)
 
-    fun dockerImageId() = "${getOrganisation(extension)}/${extension.imageName.get()}:${extension.imageTag.get()}"
+    fun dockerImageId() = project.providers.zip(getOrganisation(project, extension), getImageNameWithTag(project, extension)) { organisation, imageNameWithTag -> "$organisation/$imageNameWithTag" }
```

Comment:
I am not really sure, what the change does and what the new outcome is.Can we write a test for that? 🤔

Reply:
I checked the new behaviour locally, but yeah, I can try to add a test - not knowing how exactly it will look like...